graphql/schema/types/distro.graphql

```diff
@@ -4,3 +4,113 @@ enum DistroSettingsAccess {
   EDIT
   VIEW
 }
+
+"""
+Distro models an environment configuration for a host.
+"""
+type Distro {
```

Comment:
Should we return an `Id` as well? I feel like the front-end GraphQL linter will complain about not having an id when we fetch the distro

Reply:
Interestingly name = ID at this point. The conversion happens here: https://github.com/evergreen-ci/evergreen/blob/0dc1abcfedaaa901c7641989063a2404f77e62d1/rest/model/distro.go#L370

Reply:
oh I see...! 🤔 I don't love it since it deviates from many of our other GraphQL objects, but if that is what's already in place 🤷‍♀️